build/manifest/go.mod

```diff
@@ -0,0 +1,34 @@
+module github.com/manland/mattermost-plugin-gitlab/build/manifest
```

Comment:
If you want, you can just maintain one 'go.mod` files in the root directory. The module path would just be `module github.com/manland/mattermost-plugin-gitlab`

Reply:
Hi after testing that solution I prefer to revert because I prefer to have a clean separation between server, webapp and tools.